### ISSUE TITLE
Quick Start v2 VoiceOver: "Customize Your Site" completed cell state is not accessible

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#11493]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. If you already have a site with Quick Start setup, go to that site, double tap on "Customize Your Site", and skip to step 7.
1. Log out of the WPiOS app.
1. Go to WPiOS and double tap the "Sign up for WordPress.com" button.
1. Follow the prompts until you reach "Want a little help getting started?"
1. Double tap the "Yes, Help Me" button.
1. Try interacting with the "Completed (x)" cell on the "Customize Your Site" screen.

Result: the “Complete (x)” cell is not accessible and the completed cells state is not communicated by VoiceOver. ([7s](https://cloudup.com/cAdp2RYcvB5))

<img src="https://user-images.githubusercontent.com/1119271/56328016-2cda1d00-613a-11e9-8394-4121b700c749.png" width="270" alt="complete-cell-on-the-customize-your-site-screen">
<p style="clear:both;"><sup>Tested with WP Internal 12.2.0.20190412 on iPhone 6S iOS 12.2.</sup></p>

<p style="clear:both;"><sub>(internal reference: p77Llu-bQ6-p2 h/t @etoledom)</sub></p>
